go-controller/pkg/ovn/secondary_layer2_network_controller.go

```diff
@@ -361,6 +364,7 @@ func NewSecondaryLayer2NetworkController(
 		if err != nil {
 			return nil, fmt.Errorf("unable to create new service controller while creating new layer2 network controller: %w", err)
 		}
+		oc.defaultGatewayReconciler = kubevirt.NewDefaultGatewayReconciler(oc.watchFactory, oc.GetNetInfo(), util.GetNetworkScopedK8sMgmtHostIntfName(uint(oc.GetNetworkID())))
```

Comment:
so this reconciler is always created but only used from live migration path so application only to live migrated vms right?

Reply:
Yep, we are guarding it later on with `IsPodAllowedForMigration`